source/parser.hera

```diff
@@ -567,27 +567,26 @@ LeftHandSideExpression
 CallExpression
   # NOTE: Tracking trailing member expressions based on indentation level for implicit arguments
   "super" ArgumentsWithTrailingMemberExpressions CallExpressionRest*:rest ->
-    return {
+    return module.processGlob({
       type: "CallExpression",
       children: [$1, ...$2, ...rest.flat()],
-    }
+    })
   # Dynamic import(), with optional parentheses when not used at top level.
   # (At top level, ImportDeclaration will match first.)
   # NOTE: Using ExtendedExpression to allow for If/Switch expressions
   "import" ArgumentsWithTrailingMemberExpressions CallExpressionRest*:rest ->
-    return {
+    return module.processGlob({
```

Comment:
Not necessary for this PR but something I've been moving towards is moving the processing functions to the end of the parse ( inside processProgram ) and having the intermediates be better AST nodes which could be useful for linting/prettier type things.

Reply:
Yeah, that's probably a good idea. In this case, the paren wrapping around object literals is currently done by a `->` code block, so I stuck with that. But we could change them both together.